.github/workflows/api-e2e.yml

```diff
@@ -41,6 +45,9 @@ jobs:
         run: |
           npm ci --ignore-scripts
 
+      - name: List running containers
+        run: docker ps
+
```

Comment:
This changes will be removed, before the approve.
I used them in order to receive important information for new feature implementation.